trunk/setup.py

```diff
@@ -176,9 +181,14 @@ def import_tests():
         raise ImportError('scipy is required for this package')
     
     try:
-        import matplotlib.pyplot
+        import matplotlib
     except ImportError:

```

Comment:
We should also think about including Scipy. It is used even more than learn.

Reply:
Scipy is already tested, I realized that we were missing scikit and added it